go.mod

```diff
@@ -103,7 +102,7 @@ require (
 	go.opentelemetry.io/otel/trace v1.27.0 // indirect
 	go.opentelemetry.io/proto/otlp v1.2.0 // indirect
 	go.uber.org/multierr v1.11.0 // indirect
-	go.uber.org/zap v1.27.0 // indirect
+	go.uber.org/zap v1.27.0
```

Comment:
Can we "promote" this to the top `require` section for consistency (I thought `go mod tidy` would do this?)

Reply:
Yeah, I don't know why `go mod tidy` didn't do it--I've noticed if the `require()` blocks aren't together it tends to leave them alone and just wipe out the `// indirect` if it's already there. Bringing them into one block and then executing `go mod tidy` separates them back out as we'd expect, though.

Reply:
Yeah, I don't know why `go mod tidy` didn't do it--I've noticed if the `require()` blocks aren't together it tends to leave them alone and just wipe out the `// indirect` if it's already there. Bringing them into one block and then executing `go mod tidy` separates them back out as we'd expect, though.